src/modules/sub-items/components/table-view/table.view.component.js

```diff
@@ -78,7 +78,6 @@ export default class TableViewComponent extends Component {
             });
 
             return {
-                ...state,
```

Comment:
I know that it might be outside of the scope of this task but methods: `sortByName`, `sortByDate`, `sortByPriority` are basically identical. What changes is only sorting function. We can extract overlaping parts to another function.

I was also thinking about rewriting sort function from this:
```js
items.sort((a, b) => {
    if (isAscSort) {
        return a.content.Name.localeCompare(b.content.Name);
    }
    return b.content.Name.localeCompare(a.content.Name);
});
```
into this:
```js
items.sort((a, b) => {
    [a, b] = isAscSort ? [a, b] : [b, a];
   
    return a.content.Name.localeCompare(b.content.Name);
});
```
to reduce code duplication. I think that it is also more clear.

Sorry for this random thoughts. :)

Reply:
We may also extract swapping logic to some helper function to make it even more clear:
```js
items.sort((a, b) => {
    [a, b] = swap([a, b], isAscSort);
   
    return a.content.Name.localeCompare(b.content.Name);
});
```

Reply:
I was thinking about it yesterday and I would gladly improve this piece of code on my own in another PR if you don't mind. I would like to try using memoization and check performance. :)

Reply:
Good idea. But we don't make to make any big changes. If you feel that you will need such helpers in the future, then you're free to make such improvements. Right now, this is out of scope of this PR.